### ISSUE TITLE
new world crash 1.19

### ISSUE THREAD

**Issue by mkire** [ultixon/OccultCraft#3]
starting a new world in the  5.5 release modpack has the game CTD and a large negative number exit code. log is attached.

Game also warns of experimental settings without changing anything
[latest.log](https://github.com/ultixon/OccultCraft/files/10356002/latest.log)

**Comment by ultixon** [comment]
Have you found any way to reproduce the crash?

**Comment by Zoroark4K** [comment]
same happened to me, just created words and crash right before i had the chance to chose which spicy am I, my friend followed the same steps without crash, i don't know where this comes from

**Comment by Zoroark4K** [comment]
it doesn't happen in flat world generation , so it must be corrupt words during generation

**Comment by ultixon** [comment]
I think it's the YASD mod.
A fella in the discord ran a server and whilst his friends were in a YASD dungeon it crashed their games. I haven't ran into the crash myself however removing the mod can possibly fix it (and it'd make sense why flat worlds don't crash).

**Comment by Maxxor501** [comment]
I still had the crashing issue even when I joined a flatworld. I tried removing YASD from the pack. Not only did it not solve the problem, but removing it even made my version of Minecraft crash before entering the world unlike before where it was a second or two in-world before crashing.

This could be intuition talking over hard research (since the pack doesn't generate a crashlog when it closes my game for some reason, otherwise i'd scrub through it to give a better guess), but my hunch is it could be a rendering issue.
